charts/bandstand-web-service/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v2
 name: bandstand-web-service
-version: 2.1.2
-description: Library chart of templates for an http service
+version: 3.0.0
+description: Library chart for a web service
 type: library
```

Comment:
Need to change the chart type to `application`

Reply:
Hold your horses, trying something.

Reply:
The only way you can deploy 2 instances is with an alias, because you can't have 2 `bandstand-web-services` trees in the yaml.

From the helm docs: "A library chart is a type of Helm chart that defines chart primitives or definitions which can be shared by Helm templates in other charts. This allows users to share snippets of code that can be re-used across charts, avoiding repetition and keeping charts DRY"

This isn't a library chart, because it isn't providing snippets

Reply:
Also we need to craft a test case which deploys multiple services (e.g. how it will work in a frontend/backend situation) to actually see the impact/value of the change

Reply:
I think we need to make aws.account global (so change in [circleci-orb-bandstand](https://github.com/ktech-org/circleci-orb-bandstand/blob/main/src/jobs/install-chart.yml#L58)). It's injected by that orb but now the web service is a dependency it cannot see it. If we inject it as `global.aws.account` then the chart and all subcharts can see it as that.

Reply:
Alternatively we just add `bandstand-web-service.aws.account=$AWS_ACCOUNT_ID` to the build and leave orb, but only that chart can see it.

Reply:
I think the global is the way to go, but we'll also need to update the chart to [explicitly read from globals](https://stackoverflow.com/a/59273541/158826) for all the values we'd want to be set globally (I'm thinking things like owner, env etc. ). Unfortunately helm's handling of globals isn't very clever.